### PR TITLE
PP-4280 Reorganise Cardtypes, Tokens & Reports

### PR DESCRIPTION
**Pull request by kbottla**
## WHAT
- Organised card type related classes into `cardtype` package and sub packages
- Organised token related classes into `token` package and sub packages
- Organised performance report & transaction summary classes into `report` package and sub packages

Doesn't include

- Any changes to existing functionality

### CODE REVIEW

**Comment by mrlumbu** [comment]
Nice. The only change I would make is that I think the dashboard package imports UI concepts into connector. I would prefer it if the dashboard stuff went under `report`

**Comment by kbottla** [comment]
Thanks @mrlumbu, agree and updated.

**Review by mrlumbu — APPROVED** [review]

File: src/main/java/uk/gov/pay/connector/cardtype/model/domain/UuidAbstractEntity.java [hunk 1]
```diff
@@ -1,8 +1,13 @@
-package uk.gov.pay.connector.model.domain;
+package uk.gov.pay.connector.cardtype.model.domain;
```
Comment (alexbishop1) [inline src/main/java/uk/gov/pay/connector/cardtype/model/domain/UuidAbstractEntity.java:1]: Not sure this class wanted to move

Comment (kbottla) [reply src/main/java/uk/gov/pay/connector/cardtype/model/domain/UuidAbstractEntity.java:1]: Although the name suggests abstract, only place this is used is in CardTypeEntity and thought these can be close. But could be in `package uk.gov.pay.connector.model.domain` so something else can use it in future. Will move it in the next change

**Pull request merged**